rpcvmx/rpcvmx.go

```diff
@@ -7,6 +7,10 @@ import (
 	"github.com/vmware/vmw-guestinfo/rpcout"
 )
 
+const (
+	prefix = "guestinfo."
```

Comment:
suggest that you drop the "." from the end - we'll end up using "/" as a separator as well.
"." is the correct choice for when prepending, but shouldn't be part of the prefix check.